app/controllers/public_recipes_controller.rb

```diff
@@ -0,0 +1,20 @@
+class PublicRecipesController < ApplicationController
+  def index
+    @recipes = Recipe.order('created_at DESC').where(public: true)
+    @user = current_user
```

Comment:
- [ ] it would be better if you handle the n+1 queries by  adding `.includes(:user)`  in order to improve the performance of your app 
<img width="477" alt="Screen Shot 2022-10-08 at 12 55 35" src="https://user-images.githubusercontent.com/61790587/194706453-4e579ffa-4c14-4819-bfea-9f6106578709.png">

Reply:
Thank you for the feedback, we have implemented them now.